Generate.py

```diff
@@ -411,12 +687,13 @@ def roll_triggers(weights: dict, triggers: list, valid_keys: set) -> dict:
 def handle_option(ret: argparse.Namespace, game_weights: dict, option_key: str, option: type(Options.Option), plando_options: PlandoOptions):
```

Comment:
In addition to this breaking change not being necessary (previous comment).

It's also not a good idea to make a breaking change to a function call without changing the parameter name or type annotation.